apps/zbugs/src/zero-setup.ts

```diff
@@ -62,14 +63,24 @@ export function preload(z: Zero<Schema>) {
     .related('assignee')
     .related('labels')
     .related('viewState', q => q.where('userID', z.userID).one())
-    .related('emoji');
+    .related('emoji', emoji =>
+      emoji.related('creator', creator => creator.one()),
+    );
 
   const {cleanup, complete} = baseIssueQuery.preload();
   complete.then(() => {
     mark('preload complete');
     cleanup();
     baseIssueQuery
-      .related('comments', q => q.related('emoji').limit(10))
+      .related('comments', comments =>
+        comments
+          .related('creator', creator => creator.one())
+          .related('emoji', emoji =>
+            emoji.related('creator', creator => creator.one()),
+          )
+          .limit(INITIAL_COMMENT_LIMIT)
```

Comment:
Why do we preload this many comments @grgbkr ?

Reply:
I think 10 was the right number before at least from a ux pov. Was there some consistency thing?

Reply:
Oh I guess it has something to do with the 'load more' effect you were creating? Is it required? This seems like too aggressive of a preload.